Content.Server/Zombies/ZombieSystem.Transform.cs

```diff
@@ -292,7 +292,8 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
         _movementSpeedModifier.RefreshMovementSpeedModifiers(target);
 
         //Need to prevent them from getting an item, they have no hands.
-        // Also prevents them from becoming a Survivor. They're undead.
+        // Also prevents them from becoming a Survivor. They're undead.  
```

Comment:
```suggestion
        // Also prevents them from becoming a Survivor. They're undead.
```
let me remove the extra whitespace here.
Your IDE should complain about that with yellow squiggly lines!

Reply:
Nevermind, for some reason I can't make a commit to your PR.
Could you clean that up?